diff -r -U 3 minecraft/net/minecraft/src/CodecMus.java minecraft_patched/net/minecraft/src/CodecMus.java
--- minecraft/net/minecraft/src/CodecMus.java	2021-10-16 15:30:50.173586300 -0400
+++ minecraft_patched/net/minecraft/src/CodecMus.java	2021-10-16 15:32:48.733241500 -0400
@@ -5,6 +5,11 @@
 
 public class CodecMus extends CodecJOrbis {
     protected InputStream openInputStream() {
-        return new MusInputStream(this, this.url, this.urlConnection.getInputStream());
+        try {
+            return new MusInputStream(this, this.url, this.urlConnection.getInputStream());
+        } catch (Exception ex) {
+            ex.printStackTrace();
+            return null;
+        }
     }
 }
