diff -r -U 3 minecraft/com/mojang/minecraft/net/ConnectionThread.java minecraft_patched/com/mojang/minecraft/net/ConnectionThread.java
--- minecraft/com/mojang/minecraft/net/ConnectionThread.java	2021-11-08 21:00:30.988998800 +0200
+++ minecraft_patched/com/mojang/minecraft/net/ConnectionThread.java	2021-11-08 21:01:27.493230700 +0200
@@ -26,7 +26,6 @@
         this.username = var4;
         this.mpPass = var5;
         this.minecraft = var6;
-        super();
     }
 
     public final void run() {
diff -r -U 3 minecraft/com/mojang/minecraft/net/NetworkSkinDownloadThread.java minecraft_patched/com/mojang/minecraft/net/NetworkSkinDownloadThread.java
--- minecraft/com/mojang/minecraft/net/NetworkSkinDownloadThread.java	2021-11-08 20:58:14.080168100 +0200
+++ minecraft_patched/com/mojang/minecraft/net/NetworkSkinDownloadThread.java	2021-11-08 21:01:22.831964100 +0200
@@ -10,7 +10,6 @@
 
     NetworkSkinDownloadThread(NetworkPlayer var1) {
         this.player = var1;
-        super();
     }
 
     public final void run() {
