diff -r -U 3 minecraft/net/minecraft/src/NetworkAcceptThread.java minecraft_patched/net/minecraft/src/NetworkAcceptThread.java
--- minecraft/net/minecraft/src/NetworkAcceptThread.java	2021-10-18 16:19:15.085187600 +0300
+++ minecraft_patched/net/minecraft/src/NetworkAcceptThread.java	2021-10-18 16:23:11.756724500 +0300
@@ -21,7 +21,13 @@
             try {
                 Socket var1 = NetworkListenThread.getServerSocket(this.networkListenThread).accept();
                 if (var1 != null) {
-                    NetLoginHandler var2 = new NetLoginHandler(this.minecraftServer, var1, "Connection #" + NetworkListenThread.incrementConnections(this.networkListenThread));
+                    NetLoginHandler var2;
+					try {
+						var2 = new NetLoginHandler(this.minecraftServer, var1, "Connection #" + NetworkListenThread.incrementConnections(this.networkListenThread));
+					} catch (Exception ex) {
+						ex.printStackTrace();
+						var2 = null;
+					}
                     NetworkListenThread.addPendingConnection(this.networkListenThread, var2);
                 }
             } catch (IOException var3) {
