diff -r -U 3 minecraft/com/mojang/minecraft/server/HeartbeatThread.java minecraft_patched/com/mojang/minecraft/server/HeartbeatThread.java
--- minecraft/com/mojang/minecraft/server/HeartbeatThread.java	2021-11-03 13:22:55.220108800 +0200
+++ minecraft_patched/com/mojang/minecraft/server/HeartbeatThread.java	2021-11-03 13:25:09.524790600 +0200
@@ -12,7 +12,6 @@
     HeartbeatThread(MinecraftServer var1, HttpURLConnection var2, String var3) {
         this.urlConnection = var2;
         this.url = var3;
-        super();
     }
 
     public final void run() {
diff -r -U 3 minecraft/com/mojang/minecraft/server/MonitorBlocksThread.java minecraft_patched/com/mojang/minecraft/server/MonitorBlocksThread.java
--- minecraft/com/mojang/minecraft/server/MonitorBlocksThread.java	2021-11-03 13:22:55.224109000 +0200
+++ minecraft_patched/com/mojang/minecraft/server/MonitorBlocksThread.java	2021-11-03 13:25:13.805035400 +0200
@@ -13,7 +13,6 @@
     MonitorBlocksThread(PlayerInstance var1, byte[] var2) {
         this.netServerHandler = var1;
         this.blocks = var2;
-        super();
     }
 
     public final void run() {
diff -r -U 3 minecraft/com/mojang/minecraft/server/PlayerInstance.java minecraft_patched/com/mojang/minecraft/server/PlayerInstance.java
--- minecraft/com/mojang/minecraft/server/PlayerInstance.java	2021-11-03 13:22:55.226109200 +0200
+++ minecraft_patched/com/mojang/minecraft/server/PlayerInstance.java	2021-11-03 13:26:11.064310500 +0200
@@ -70,6 +70,7 @@
             byte var5;
             short var9;
             short var12;
+            short var10000;
             short var10001;
             short var10002;
             short var10003;
@@ -136,7 +137,7 @@
 
                 return;
             } else if (var1 == Packet.PLAYER_TELEPORT && this.sendingPackets) {
-                (Byte)var2[0];
+            	var10000 = (Byte)var2[0];
                 var10001 = (Short)var2[1];
                 var10002 = (Short)var2[2];
                 var10003 = (Short)var2[3];
