diff -r -U 3 minecraft/com/mojang/rubydung/level/Chunk.java minecraft_patched/com/mojang/rubydung/level/Chunk.java
--- minecraft/com/mojang/rubydung/level/Chunk.java	2021-10-30 15:14:42.129054100 +0300
+++ minecraft_patched/com/mojang/rubydung/level/Chunk.java	2021-10-30 15:09:42.600922100 +0300
@@ -47,7 +47,7 @@
                 for(int y = this.y0; y < this.y1; ++y) {
                     for(int z = this.z0; z < this.z1; ++z) {
                         if (this.level.isTile(x, y, z)) {
-                            int tex = y != this.level.depth * 2 / 3;
+                            boolean tex = y != this.level.depth * 2 / 3;
                             ++tiles;
                             if (!tex) {
                                 Tile.rock.render(t, this.level, layer, x, y, z);
