diff -r -U 3 minecraft/com/mojang/minecraft/server/HeartbeatThread.java minecraft_patched/com/mojang/minecraft/server/HeartbeatThread.java
--- minecraft/com/mojang/minecraft/server/HeartbeatThread.java	2021-11-08 15:07:32.598073000 +0200
+++ minecraft_patched/com/mojang/minecraft/server/HeartbeatThread.java	2021-11-08 15:17:57.398809500 +0200
@@ -17,7 +17,6 @@
     HeartbeatThread(MinecraftServer var1, String var2) {
         this.minecraft = var1;
         this.content = var2;
-        super();
     }
 
     public final void run() {
diff -r -U 3 minecraft/com/mojang/minecraft/server/MonitorBlocksThread.java minecraft_patched/com/mojang/minecraft/server/MonitorBlocksThread.java
--- minecraft/com/mojang/minecraft/server/MonitorBlocksThread.java	2021-11-08 15:07:32.603073300 +0200
+++ minecraft_patched/com/mojang/minecraft/server/MonitorBlocksThread.java	2021-11-08 15:17:54.864664600 +0200
@@ -13,7 +13,6 @@
     MonitorBlocksThread(PlayerInstance var1, byte[] var2) {
         this.playerInstance = var1;
         this.blocks = var2;
-        super();
     }
 
     public final void run() {
diff -r -U 3 minecraft/com/mojang/minecraft/server/PlayerInstance.java minecraft_patched/com/mojang/minecraft/server/PlayerInstance.java
--- minecraft/com/mojang/minecraft/server/PlayerInstance.java	2021-11-08 15:07:32.607073500 +0200
+++ minecraft_patched/com/mojang/minecraft/server/PlayerInstance.java	2021-11-08 15:17:51.358464000 +0200
@@ -121,7 +121,7 @@
                     return;
                 } else if (var1 == Packet.PLAYER_TELEPORT && this.sendingPackets) {
                     PlayerInstance var10000 = this;
-                    (Byte)var2[0];
+                    short var1000 = (Byte)var2[0];
                     short var10001 = (Short)var2[1];
                     short var10002 = (Short)var2[2];
                     short var10003 = (Short)var2[3];
