diff -r -U 3 minecraft/net/minecraft/client/CanvasMinecraftApplet.java minecraft_patched/net/minecraft/client/CanvasMinecraftApplet.java
--- minecraft/net/minecraft/client/CanvasMinecraftApplet.java	2021-10-17 19:24:58.613398000 -0400
+++ minecraft_patched/net/minecraft/client/CanvasMinecraftApplet.java	2021-10-17 19:04:23.280735000 -0400
@@ -8,7 +8,6 @@
 
     CanvasMinecraftApplet(MinecraftApplet var1) {
         this.mcApplet = var1;
-        super();
     }
 
     public final synchronized void addNotify() {
diff -r -U 3 minecraft/net/minecraft/client/gui/GuiLevelDialog.java minecraft_patched/net/minecraft/client/gui/GuiLevelDialog.java
--- minecraft/net/minecraft/client/gui/GuiLevelDialog.java	2021-10-17 19:24:58.679221600 -0400
+++ minecraft_patched/net/minecraft/client/gui/GuiLevelDialog.java	2021-10-17 19:04:23.346553900 -0400
@@ -9,7 +9,6 @@
 
     GuiLevelDialog(GuiLoadLevelScreen var1) {
         this.screen = var1;
-        super();
     }
 
     public final void run() {
diff -r -U 3 minecraft/net/minecraft/client/gui/GuiLoadLevelScreen.java minecraft_patched/net/minecraft/client/gui/GuiLoadLevelScreen.java
--- minecraft/net/minecraft/client/gui/GuiLoadLevelScreen.java	2021-10-17 19:24:58.691189300 -0400
+++ minecraft_patched/net/minecraft/client/gui/GuiLoadLevelScreen.java	2021-10-17 19:17:13.905850600 -0400
@@ -15,7 +15,7 @@
     private GuiScreen parent;
     private boolean finished = false;
     private boolean loaded = false;
-    private String[] levels = null;
+    private String[] levels = {"Load From Button","Load From Button","Load From Button","Load From Button","Load From Button"};
     private String status = "";
     protected String title = "Load level";
     private boolean frozen = false;
@@ -41,9 +41,9 @@
     public void run() {
         try {
             this.status = "Getting level list..";
-            URL var1 = new URL("http://" + this.mc.minecraftUri + "/listmaps.jsp?user=" + this.mc.session.username);
+            /*URL var1 = new URL("http://" + this.mc.minecraftUri + "/listmaps.jsp?user=" + this.mc.session.username);
             BufferedReader var3 = new BufferedReader(new InputStreamReader(var1.openConnection().getInputStream()));
-            this.levels = var3.readLine().split(";");
+            this.levels = var3.readLine().split(";");*/
             if (this.levels.length >= 5) {
                 this.setLevels(this.levels);
                 this.loaded = true;
diff -r -U 3 minecraft/net/minecraft/client/gui/SlotArmor.java minecraft_patched/net/minecraft/client/gui/SlotArmor.java
--- minecraft/net/minecraft/client/gui/SlotArmor.java	2021-10-17 19:24:58.709142500 -0400
+++ minecraft_patched/net/minecraft/client/gui/SlotArmor.java	2021-10-17 19:04:23.380439600 -0400
@@ -9,8 +9,8 @@
     private int armorType;
 
     SlotArmor(GuiInventory var1, GuiContainer var2, IInventory var3, int var4, int var5, int var6, int var7) {
-        this.armorType = var7;
-        super(var2, var3, var4, 8, var6);
+    	super(var2, var3, var4, 8, var6);
+    	this.armorType = var7;
     }
 
     public final boolean isItemValid(ItemStack var1) {
diff -r -U 3 minecraft/net/minecraft/client/Minecraft.java minecraft_patched/net/minecraft/client/Minecraft.java
--- minecraft/net/minecraft/client/Minecraft.java	2021-10-17 19:24:58.623370900 -0400
+++ minecraft_patched/net/minecraft/client/Minecraft.java	2021-10-17 19:21:56.042669800 -0400
@@ -98,7 +98,7 @@
     private String server;
     private TextureWaterFX textureWaterFX;
     private TextureLavaFX textureLavaFX;
-    volatile boolean running;
+    public volatile boolean running;
     public String debug;
     public boolean inventoryScreen;
     private int prevFrameTime;
@@ -791,7 +791,7 @@
     public final void generateLevel(int var1, int var2, int var3, int var4) {
         this.setLevel((World)null);
         System.gc();
-        String var5 = this.session != null ? this.session.username : "anonymous";
+        final String var5 = (this.session != null) ? this.session.username : "anonymous";
         LevelGenerator var6;
         (var6 = new LevelGenerator(this.loadingScreen)).islandGen = var3 == 1;
         var6.floatingGen = var3 == 2;
@@ -817,11 +817,11 @@
         }
 
         try {
-            BufferedReader var2;
-            Integer.parseInt((var2 = new BufferedReader(new InputStreamReader((new URL(this.mcApplet.getDocumentBase() + "?n=" + this.session.username + "&i=" + this.session.sessionId)).openStream()))).readLine());
-            var2.close();
-            if (this.mcApplet.getDocumentBase().toString().startsWith("http://www.minecraft.net/") || this.mcApplet.getDocumentBase().toString().startsWith("http://minecraft.net/")) {
-                this.theWorld = var1;
+            //BufferedReader var2;
+            //Integer.parseInt((var2 = new BufferedReader(new InputStreamReader((new URL(this.mcApplet.getDocumentBase() + "?n=" + this.session.username + "&i=" + this.session.sessionId)).openStream()))).readLine());
+            //var2.close();
+            if (!this.mcApplet.getDocumentBase().toString().startsWith("http://www.minecraft.net/") || !this.mcApplet.getDocumentBase().toString().startsWith("http://minecraft.net/")) {
+               this.theWorld = var1;
             }
         } catch (Throwable var3) {
         }
diff -r -U 3 minecraft/net/minecraft/client/MinecraftApplet.java minecraft_patched/net/minecraft/client/MinecraftApplet.java
--- minecraft/net/minecraft/client/MinecraftApplet.java	2021-10-17 19:24:58.624368000 -0400
+++ minecraft_patched/net/minecraft/client/MinecraftApplet.java	2021-10-17 19:17:49.778945500 -0400
@@ -24,8 +24,8 @@
             var10002.minecraftUri = var10000.append(var10002.minecraftUri).append(":").append(this.getDocumentBase().getPort()).toString();
         }
 
-        if (this.getParameter("username") != null && this.getParameter("sessionid") != null) {
-            this.mc.session = new SessionData(this.getParameter("username"), this.getParameter("sessionid"));
+        if (this.getParameter("username") == null && this.getParameter("sessionid") == null) {
+        	this.mc.session = new SessionData(this.getParameter("username"), this.getParameter("sessionid"));
             if (this.getParameter("mppass") != null) {
                 this.getParameter("mppass");
             }
diff -r -U 3 minecraft/net/minecraft/client/player/PlayerControllerCreative.java minecraft_patched/net/minecraft/client/player/PlayerControllerCreative.java
--- minecraft/net/minecraft/client/player/PlayerControllerCreative.java	2021-10-17 19:24:58.792348400 -0400
+++ minecraft_patched/net/minecraft/client/player/PlayerControllerCreative.java	2021-10-17 19:04:23.471661000 -0400
@@ -1,5 +1,6 @@
 package net.minecraft.client.player;
 
+import net.minecraft.client.Minecraft;
 import net.minecraft.client.SessionData;
 import net.minecraft.game.block.Block;
 import net.minecraft.game.entity.player.EntityPlayer;
@@ -8,7 +9,11 @@
 import net.minecraft.game.level.World;
 
 public final class PlayerControllerCreative extends PlayerController {
-    private MobSpawner mobSpawner;
+    public PlayerControllerCreative(Minecraft client) {
+		super(client);
+	}
+
+	private MobSpawner mobSpawner;
 
     public final void blankPlayerFunction(EntityPlayer var1) {
         for(int var2 = 0; var2 < 9; ++var2) {
diff -r -U 3 minecraft/net/minecraft/client/renderer/entity/RenderPlayer.java minecraft_patched/net/minecraft/client/renderer/entity/RenderPlayer.java
--- minecraft/net/minecraft/client/renderer/entity/RenderPlayer.java	2021-10-17 19:24:58.874370900 -0400
+++ minecraft_patched/net/minecraft/client/renderer/entity/RenderPlayer.java	2021-10-17 19:04:23.561374700 -0400
@@ -41,7 +41,7 @@
         if ((var6 = var5.inventory.armorInventory[var4]) != null && (var7 = var6.getItem()) instanceof ItemArmor) {
             ItemArmor var8 = (ItemArmor)var7;
             this.loadTexture("/armor/" + armorFilenamePrefix[var8.renderIndex] + "_" + (var3 == 2 ? 2 : 1) + ".png");
-            ModelBiped var9;
+            ModelBiped var9 = null;
             (var9 = var3 == 2 ? this.modelArmor : this.modelArmorChestplate).bipedHead.showModel = var3 == 0;
             var9.bipedHeadwear.showModel = var3 == 0;
             var9.bipedBody.showModel = var3 == 1 || var3 == 2;
diff -r -U 3 minecraft/net/minecraft/client/renderer/ThreadDownloadImage.java minecraft_patched/net/minecraft/client/renderer/ThreadDownloadImage.java
--- minecraft/net/minecraft/client/renderer/ThreadDownloadImage.java	2021-10-17 19:24:58.834236900 -0400
+++ minecraft_patched/net/minecraft/client/renderer/ThreadDownloadImage.java	2021-10-17 19:04:23.518236800 -0400
@@ -16,7 +16,6 @@
         this.imageData = var1;
         this.location = var2;
         this.buffer = var3;
-        super();
     }
 
     public final void run() {
diff -r -U 3 minecraft/net/minecraft/game/item/Item.java minecraft_patched/net/minecraft/game/item/Item.java
--- minecraft/net/minecraft/game/item/Item.java	2021-10-17 19:24:59.009785100 -0400
+++ minecraft_patched/net/minecraft/game/item/Item.java	2021-10-17 19:04:23.703625000 -0400
@@ -76,7 +76,7 @@
     public final int shiftedIndex;
     protected int maxStackSize = 64;
     protected int maxDamage = 32;
-    private int iconIndex;
+    protected int iconIndex;
 
     protected Item(int var1) {
         this.shiftedIndex = var1 + 256;
diff -r -U 3 minecraft/net/minecraft/game/level/BlockMapSlot.java minecraft_patched/net/minecraft/game/level/BlockMapSlot.java
--- minecraft/net/minecraft/game/level/BlockMapSlot.java	2021-10-17 19:24:59.039676500 -0400
+++ minecraft_patched/net/minecraft/game/level/BlockMapSlot.java	2021-10-17 19:04:23.733518900 -0400
@@ -11,7 +11,6 @@
 
     private BlockMapSlot(BlockMap var1, byte var2) {
         this.blockMap = var1;
-        super();
     }
 
     public final BlockMapSlot init(float var1, float var2, float var3) {
diff -r -U 3 minecraft/net/minecraft/game/level/LevelLoader.java minecraft_patched/net/minecraft/game/level/LevelLoader.java
--- minecraft/net/minecraft/game/level/LevelLoader.java	2021-10-17 19:24:59.042669300 -0400
+++ minecraft_patched/net/minecraft/game/level/LevelLoader.java	2021-10-17 19:04:23.736546400 -0400
@@ -43,8 +43,13 @@
             this.guiLoading.setText("Reading..");
         }
 
-        NBTTagCompound var13;
-        NBTTagCompound var2 = (var13 = GuiLevelLoading.writeLevelTags(var1)).getCompoundTag("About");
+        NBTTagCompound var13 = null;
+        NBTTagCompound var2 = null;
+		try {
+			var2 = (var13 = GuiLevelLoading.writeLevelTags(var1)).getCompoundTag("About");
+		} catch (IOException ex) {
+			ex.printStackTrace();
+		}
         NBTTagCompound var3 = var13.getCompoundTag("Map");
         NBTTagCompound var4 = var13.getCompoundTag("Environment");
         NBTTagList var5 = var13.getTagList("Entities");
