diff -r -U 3 minecraft/com/mojang/minecraft/server/HeartbeatThread.java minecraft_patched/com/mojang/minecraft/server/HeartbeatThread.java
--- minecraft/com/mojang/minecraft/server/HeartbeatThread.java	2021-11-08 20:58:18.221404900 +0200
+++ minecraft_patched/com/mojang/minecraft/server/HeartbeatThread.java	2021-11-08 21:00:20.702410500 +0200
@@ -17,7 +17,6 @@
     HeartbeatThread(MinecraftServer var1, String var2) {
         this.minecraft = var1;
         this.content = var2;
-        super();
     }
 
     public final void run() {
diff -r -U 3 minecraft/com/mojang/minecraft/server/MonitorBlocksThread.java minecraft_patched/com/mojang/minecraft/server/MonitorBlocksThread.java
--- minecraft/com/mojang/minecraft/server/MonitorBlocksThread.java	2021-11-08 20:58:18.226405200 +0200
+++ minecraft_patched/com/mojang/minecraft/server/MonitorBlocksThread.java	2021-11-08 21:00:17.559230700 +0200
@@ -13,7 +13,6 @@
     MonitorBlocksThread(PlayerInstance var1, byte[] var2) {
         this.playerInstance = var1;
         this.blocks = var2;
-        super();
     }
 
     public final void run() {
diff -r -U 3 minecraft/com/mojang/minecraft/server/PlayerInstance.java minecraft_patched/com/mojang/minecraft/server/PlayerInstance.java
--- minecraft/com/mojang/minecraft/server/PlayerInstance.java	2021-11-08 20:58:18.230405500 +0200
+++ minecraft_patched/com/mojang/minecraft/server/PlayerInstance.java	2021-11-08 21:00:10.605833000 +0200
@@ -95,7 +95,7 @@
 
                 } else {
                     if (var1 == Packet.PLAYER_TELEPORT) {
-                        (Byte)var2[0];
+                    	short var10000 = (Byte)var2[0];
                         short var10001 = (Short)var2[1];
                         short var10002 = (Short)var2[2];
                         short var10003 = (Short)var2[3];
