diff -r -U 3 minecraft/net/minecraft/src/TileEntityChest.java minecraft_patched/net/minecraft/src/TileEntityChest.java
--- minecraft/net/minecraft/src/TileEntityChest.java	2021-11-06 19:52:16.922692500 +0200
+++ minecraft_patched/net/minecraft/src/TileEntityChest.java	2021-11-06 20:00:22.247451500 +0200
@@ -88,4 +88,7 @@
             return var1.getDistanceSq((double)this.xCoord + 0.5D, (double)this.yCoord + 0.5D, (double)this.zCoord + 0.5D) <= 64.0D;
         }
     }
+
+	public void onInventoryChanged() {
+	}
 }
diff -r -U 3 minecraft/net/minecraft/src/TileEntityDispenser.java minecraft_patched/net/minecraft/src/TileEntityDispenser.java
--- minecraft/net/minecraft/src/TileEntityDispenser.java	2021-11-06 19:52:16.924692600 +0200
+++ minecraft_patched/net/minecraft/src/TileEntityDispenser.java	2021-11-06 20:00:21.525410200 +0200
@@ -109,4 +109,7 @@
             return var1.getDistanceSq((double)this.xCoord + 0.5D, (double)this.yCoord + 0.5D, (double)this.zCoord + 0.5D) <= 64.0D;
         }
     }
+
+	public void onInventoryChanged() {
+	}
 }
diff -r -U 3 minecraft/net/minecraft/src/TileEntityFurnace.java minecraft_patched/net/minecraft/src/TileEntityFurnace.java
--- minecraft/net/minecraft/src/TileEntityFurnace.java	2021-11-06 19:52:16.926692800 +0200
+++ minecraft_patched/net/minecraft/src/TileEntityFurnace.java	2021-11-06 20:00:20.220335600 +0200
@@ -194,4 +194,7 @@
             return var1.getDistanceSq((double)this.xCoord + 0.5D, (double)this.yCoord + 0.5D, (double)this.zCoord + 0.5D) <= 64.0D;
         }
     }
+
+	public void onInventoryChanged() {
+	}
 }
diff -r -U 3 minecraft/net/minecraft/src/EntityCreeper.java minecraft_patched/net/minecraft/src/EntityCreeper.java
--- minecraft/net/minecraft/src/EntityCreeper.java	2021-11-06 22:37:59.981403000 +0200
+++ minecraft_patched/net/minecraft/src/EntityCreeper.java	2021-11-06 22:37:20.825163300 +0200
@@ -11,7 +11,7 @@
 
     protected void entityInit() {
         super.entityInit();
-        this.dataWatcher.addObject(16, -1);
+        this.dataWatcher.addObject(16, (byte)-1);
     }
 
     public void writeEntityToNBT(NBTTagCompound var1) {
